mayastor/src/bin/mayastor-client/nexus_cli.rs

```diff
@@ -54,6 +54,13 @@ pub fn subcommands<'a, 'b>() -> App<'a, 'b> {
                 .help("uuid for the nexus"),
         );
 
+    let set_ana_state = SubCommand::with_name("set_ana_state")
```

Comment:
```suggestion
    let set_ana_state = SubCommand::with_name("ana")
```

Reply:
I've opted for `ana_state` as `ana` on its own is a little vague.